platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/editor/StateEditorSave.java

```diff
@@ -60,8 +67,18 @@ public void run() {
                 Files.createDirectories(directory);
             }
 
+            final ConfigSource settingsConfigSource = LegacyFileConfigSource.ofSettingsFile();
+            final ConfigSource mappedSettingsConfigSource = ConfigMappings.addConfigMapping(settingsConfigSource);
+            final ConfigSource threadCountPropertyConfigSource = new ThreadCountPropertyConfigSource();
+
+            final ConfigurationBuilder configurationBuilder = ConfigurationBuilder.create()
+                    .withSource(mappedSettingsConfigSource)
+                    .withSource(threadCountPropertyConfigSource);
+            ConfigUtils.scanAndRegisterAllConfigTypes(configurationBuilder, Set.of("com.swirlds"));
```

Comment:
we should extract the "default config setup" in a util method

Reply:
Introduced `DefaultConfiguration`